tutorial/index.jade

```diff
@@ -1,10 +1,155 @@
+
+  .step(data-title='The Form Template', data-file='views/users/create.jade', data-language='jade')
+    :markdown
+      ## The Form Template
+
+      When we create a new user, we ask for a name, e-mail address, and a password. 
+      In order to validate their account we also ask them for a mobile number with a country code. 
+      We use Authy to send them a one-time password via SMS to this phone number.
+     
+      Now the user is logged in but not verified. In the next steps we'll learn how to verify the user using authy.
+
+  .step(data-title="Configuring Authy", data-file="config.js", data-highlight="24-26")
+    :markdown
+      ## Configuring Authy
+
+      In `config.js`, we list configuration parameters for the application. Most are pulled in from system environment variables, which is a helpful way to access sensitive values (like API keys). This prevents us from accidentally checking them in to source control.
+
+      Now, we need our Authy production key ([sign up for Authy here](https://dashboard.authy.com/signup)). When you create an Authy application, the production key is found on the dashboard:
+
+      ![Authy Dashboard](//s3.amazonaws.com/howtodocs/2fa-authy-dashboard.png)
+
+      Next, we need to jump over to the `User` model to configure the Authy client and create an instance method to send a one-time password.
+
+  .step(data-title='Send a Verification Token', data-file='models/User.js', data-highlight='73-101')
+    :markdown
+      ## Send a Verification Token
+
+      When it comes time to actually send the user a verification code, we do that in our model function here.
+
+      Before sending the code, an [Authy user](https://docs.authy.com/#section-Enabling_two-factor_on_a_user) needs to exist as a pair for our `User` model in the database.  If the `authyId` for our user instance hasn't been set, we use the Authy API client to create an associated Authy user and store that ID in our database.
+
+      Once the user has an `authyId`, we can actually send a verification code to that user's mobile phone.
+
+      ---
+
+      **See Also:**
+      * [Node Client for Authy API](//github.com/authy/node-authy)
+
+  .step(data-title='Verifying the Code: Controller', data-file='controllers/users.js', data-highlight='95-147')
+    :markdown
+      ## Verifying the Code: Controller
+
+      This controller function handles the form submission. It's a little longer than the others, but it has a lot to do. It needs to:
+
+      * Load a `User` model for the current verification request
+      * Use an instance function on the model object to verify the code that was entered by the user
+      * If the code entered was valid, flip a boolean flag on the user model to indicate the account was verified
+
+      Let's look at the `User` model to check out the instance method that handles verifying the code with Authy next.
```

Comment:
@ronualdo what about adding some reference to the [`Authy#verify`](https://www.npmjs.com/package/authy#verify-token) method. This one is used under the hood at `verifyAuthyToken`.

Also adding a reference to [`Model#save`](http://mongoosejs.com/docs/api.html#model_Model-save) would be good. 

I'd appreciate hearing your thoughts on this.